src/main/java/org/gridsuite/study/server/dto/TombstonedEquipmentInfos.java

```diff
@@ -22,7 +22,7 @@
 @ToString(callSuper = true)
 @EqualsAndHashCode(callSuper = true)
 @Schema(description = "Tombstoned equipment infos")
-@Document(indexName = "#{@environment.getProperty('index.prefix')}tombstoned-equipments")
+@Document(indexName = "#{@environment.getProperty('powsybl-ws.elasticsearch.index.prefix')}tombstoned-equipments")
```

Comment:
This seems a completly different scope. Maybe we should commit this fix in another PR ?

Reply:
Removed